### PR TITLE
Add initial product schema yaml file

### PR DESCRIPTION
**Pull request by RyanLassigBanks**
Signed-off-by: Ryan Banks <rbanks@bitwise.io>

### CODE REVIEW

**Review by dnewh — APPROVED** [review]

**Review by dnewh — CHANGES_REQUESTED** [review]

**Review by dplumb94 — APPROVED** [review]

File: cli/schemas/product.yaml [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+    - name: packaging_depth_uom
+      data_type: Enum
+      description: The unit of measure for packaging depth.
+      enum_options: [FOT, INH, MTR]
+      required: false
+    - name: packaging_height
+      data_type: Number
+      description: |
+        The height of the packaging as measured according to the GDSN Package
+        Measurement Rules. If the packaging dimensions is for a platform or
```
Comment (rbuysse) [inline cli/schemas/product.yaml:255]: s/is/are

File: cli/schemas/product.yaml [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+    - name: packaging_type_code
+      data_type: Number
+      description: ""
+      number_exponent: 0
+      required: false
+    - name: packaging_width
+      data_type: Number
+      description: |
+        The width of the packaging as measured according to the GDSN Package
+        Measurement Rules. If the packaging dimensions is for a platform or
```
Comment (rbuysse) [inline cli/schemas/product.yaml:275]: s/is/are

File: cli/schemas/product.yaml [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+    - name: sequence_number
+      data_type: Number
+      description: |
+        A number which specifies a single overall sequence of contributor names.
+      number_exponent: 0
+      required: false
+    - name: serving_size_description
+      data_type: String
+      description: |
+        "A free text field specifying the serving size for which the nutrient
```
Comment (rbuysse) [inline cli/schemas/product.yaml:343]: Is this meant to have quotes around it?

File: cli/schemas/product.yaml [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+      required: false
+    - name: brand_name
+      data_type: String
+      description: |
+        The recognisable name used by a brand owner to uniquely identify a
+        line of trade item or services. This is recognizable by the consumer.
+      required: false
+    - name: colour_description
+      data_type: String
+      description: What is the color of the package? E.g. Green or White
```
Comment (rbuysse) [inline cli/schemas/product.yaml:50]: Can this be re-written as a statement? This is the only description posed as a question.

File: cli/schemas/product.yaml (hunk 3 above)
Comment (rbuysse) [inline cli/schemas/product.yaml:343]: Is this meant to have quotes around it?

File: cli/schemas/product.yaml [hunk 5]
```diff
@@ -0,0 +1,474 @@
...
+      required: false
+    - name: nutritional_claim
+      data_type: String
+      description: Free text field for any additional nutritional claims.
+      required: false
+    - name: packaging_depth
+      data_type: Number
+      description: |
+        The depth of the packaging as measured according to the GDSN Package
+        Measurement Rules. If the packaging dimensions is for a platform or
```
Comment (rbuysse) [inline cli/schemas/product.yaml:240]: There's a subject-verb disagreement here.

File: cli/schemas/product.yaml (hunk 1 above)
Comment (rbuysse) [inline cli/schemas/product.yaml:255]: Subject-verb disagreement

File: cli/schemas/product.yaml [hunk 6]
```diff
@@ -0,0 +1,474 @@
...
+    - name: trade_item_group_identification_code_reference
+      data_type: String
+      description: |
+        A code assigned by the supplier or manufacturer to logically group trade
+        item independently from the Global trade item Classification.
+      required: false
+    - name: trade_item_key_words
+      data_type: String
+      description: |
+        Words or phrases that enables web search engines to find trade items on
```
Comment (rbuysse) [inline cli/schemas/product.yaml:379]: Subject-verb disagreement.

File: cli/schemas/product.yaml [hunk 7]
```diff
@@ -0,0 +1,474 @@
...
+        the internet for example Shampoo, Lather, Baby.
```
Comment (rbuysse) [inline cli/schemas/product.yaml:380]: colon after example

File: cli/schemas/product.yaml [hunk 8]
```diff
@@ -0,0 +1,474 @@
...
+    - name: may_contain_allergen_list
+      data_type: String
+      description: |
+        A list composed of the allergens that may be contained in the product
+      required: false
+    - name: key_words_list
+      data_type: String
+      description: A list composed of the key words
+      required: false
+    - name: "features_and_benefits_list"
```
Comment (rbuysse) [inline cli/schemas/product.yaml:471]: Is this meant to have quotes around it?

File: cli/schemas/product.yaml [hunk 9]
```diff
@@ -0,0 +1,474 @@
...
+      data_type: Enum
+      description: |
+        A code depicting the type of activity being performed on a trade item
+        for example ASSEMBLY, BOTTLING, LAST_PROCESSING.
+      enum_options: [ASSEMBLY, BIRTH, BOTTLING, CATCH_ZONE, FARMING,
+        IMPORTATION, LAST_PROCESSING, MILKING, REARING, SLAUGHTER]
+      required: false
+    - name: country_of_activity
+      data_type: String
+      description: "The geographic area where an activity has taken place."
```
Comment (rbuysse) [inline cli/schemas/product.yaml:438]: Is this meant to have quotes around it?

**Review by rbuysse — CHANGES_REQUESTED** [review]

**Review by rbuysse — DISMISSED** [review]

**Review by vaporos — CHANGES_REQUESTED** [review]
Need better attribution.

**Review by vaporos — DISMISSED** [review]

**Review by dplumb94 — DISMISSED** [review]

**Comment by RyanLassigBanks** [comment]
We've decided to take the product schema in a different direction that makes this PR obsolete

**Pull request closed without merging**